index.js

```diff
@@ -46,6 +50,14 @@ document.getElementById('submit_button').addEventListener('click', function () {
     user_feedback_elem.style.display = 'block'
 })
 
+document.getElementById('next_question_button').addEventListener('click', function () {
+    document.getElementById('reveal_button').disabled = false
```

Comment:
Might be worth clearing the answer box when you hit next? If you have something entered, then you reveal and hit next, the text is still there